docs/general/accessibility.md

```diff
@@ -0,0 +1,39 @@
+# Accessible Charts
+
+Chart.js charts are rendered on user provided `canvas` elements. Thus it is up to the user to create the `canvas` element in a way that is accessible. The `canvas` element has support in all browsers and will render on screen but the `canvas` content will not be accessible to screen readers.
```

Comment:
nit: it should be `on a user provided`

Reply:
I used plural charts and plural elements. I believe that is grammatically correct.